resources/istio-kyma-patch/README.md

```diff
@@ -0,0 +1,14 @@
+# Istio Kyma patch
+
+## Overview
+
+This chart packs [patch script](../../components/istio-kyma-patch/README.md) as kubernetes job.
+
+## Patches
```

Comment:
This is the Charts template:
https://github.com/kyma-project/community/blob/master/guidelines/templates/resources/chart_README.md

I would remove this heading.